en/flight_modes/return.md

```diff
@@ -2,33 +2,107 @@
+At the end there are sections explaining the *default* (preconfigured) behaviour for each [vehicle type](#default_configuration).
 
-<span></span>
 > **Note** 
->  * This mode requires GPS.
+> * This mode requires GPS.
 > * This mode is automatic - no user intervention is *required* to control the vehicle.
 > * RC control switches can be used to change flight modes on any vehicle.
     The effect of RC stick movement depends on the vehicle type.
 
-The specific behaviour for each vehicle type is described below.
 
-## Multi-Copter (MC)
+## Return Types (RTL_TYPE) {#return_types}
+
+PX4 allows you to specify one of three different approaches for finding an unobstructed path to a safe destination and/or landing:
+- [Safety point return](#safety_point_return) ([RTL_TYPE=0](#RTL_TYPE)): Ascend to safe altitude and return to a safe point (home location)
+- [Mission landing return](#mission_landing_return) (`RTL_TYPE=1`): Ascend to a safe altitude, fly direct to start of mission landing (if defined) and land.
+- [Mission path return](#mission_path_return) (`RTL_TYPE=2`): Use mission path and continue to mission landing (if defined). If no mission landing defined, reverse mission to home.
+
+For the mission based return types, the vehicle will fall back to a safe point return if no mission/mission landing is defined.
+
+More detailed explanations for each of the types are provided in the following sections.
+
+### Safety Point Return Type (RTL_TYPE=0) {#safety_point_return}
+
+In this return type the vehicle:
+- Ascends to a safe [return altitude](#return_altitude) (above any expected obstacles).
+- Flies to safe point (home position) via direct path
+- [Land or waits](#arrival) at descent altitude
+
+> **Note** This return type is the fallback for the other types if no mission or mission landing is defined.
+
+<span></span>
+> **Note** This return type is commonly referred to as *Return to Launch* (RTL) and *Return to Home* (RTH).
+
+
+### Mission Landing Return Type (RTL_TYPE=1) {#mission_landing_return}
+
+This return type uses a *mission landing pattern* (if defined) to set the return destination and landing behaviour.
+
+> **Note** A mission landing pattern consists of a [MAV_CMD_DO_LAND_START](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_LAND_START), one or more position waypoints, and a [MAV_CMD_NAV_LAND](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_LAND).
 
-A copter/multi-rotor will first ascend to the return altitude (*by default*, [RTL_RETURN_ALT](#RTL_RETURN_ALT)) and then fly to the home position in a straight line and constant altitude (if already above the return altitude it will return at its current altitude).
+If a mission landing has been defined, the vehicle will:
+- Ascend to a safe [return altitude](#return_altitude) above any expected obstacles.
+- Fly a direct path to the start of a mission landing sequence
+- Descend/ascend to the specified waypoint altitude
```

Comment:
Would be good if this could be confirmed (ie that we fly to safe altitude), then to start of landing sequence, descend to its altitude, then continue.

Reply:
@hamishwillee Yes, that sounds correct.